lib/build-secrets/index.ts

```diff
@@ -129,14 +126,10 @@ export async function populateSecrets(
 		},
 		JSON.stringify(secrets),
 	);
-	pack.entry(
-		{ name: 'populate.sh' },
-		await fs.readFile(path.join(__dirname, 'populate.sh')),
-	);
 	const dockerfileContent = dockerfileTemplate.process(
 		await fs.readFile(path.join(__dirname, './Dockerfile'), 'utf8'),
 		{
-			ARCH: getAlpineArch(architecture),
+			ARCH: architecture,
```

Comment:
The current idea is that the images generated and uploaded to Docker Hub by `balena-io-modules/balena-multibuild-scripts` will be prefixed with the architecture values returned by `docker.version().Arch` (i.e. one of 'arm', 'arm64', '386', 'amd64'), so there's no need for the `getAlpineArch()` logic.